### ISSUE TITLE
Docs feedback: /configuration/gcs.md

### ISSUE THREAD

**Issue by julien-sugg** [grafana/tempo#722]
Greetings,

# tl;dr

The permission `storage.buckets.get` is missing in the `configuration/gcs.md` documentation in order to apply the `principle of least privilege` using GCS as the storage backend.

# Details

I started playing with Tempo today using my `Google Kubernetes Engine` cluster with `GCS + Workload Identity` enabled.

I applied the least `principle of least privilege` and genuinly created a custom role with only the mentionned permissions. However, it appears that the required permissions specified in the `configuration/gcs.md` documentation are incomplete as I was getting the following error in the logs:

*extract of the querier logs*:
```bash
level=info ts=2021-06-01T13:32:01.351844132Z caller=main.go:89 msg="Starting Tempo" version="(version=0.7.0, branch=HEAD, revision=83021a11e)"
level=error ts=2021-06-01T13:32:01.516687395Z caller=main.go:92 msg="error running Tempo" err="failed to init module services error initialising module: store: failed to create store getting bucket attrs: googleapi: Error 403: **redacted**@**redacted**.iam.gserviceaccount.com does not have **storage.buckets.get** access to the Google Cloud Storage bucket., forbidden"
```

Once I added the `storage.buckets.get` permission in the custom role, everything worked like a charm.

**Comment by joe-elliott** [comment]
Thanks for the heads up!
